Sources_v3/Workers/Background/NewUserQueryUpdater_Tests.swift

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient
+import XCTest
+
+class NewUserQueryUpdater_Tests: StressTestCase {
+    typealias ExtraData = DefaultExtraData.User
+    
+    private var env: TestEnvironment!
+    
+    var database: DatabaseContainerMock!
+    var webSocketClient: WebSocketClientMock!
+    var apiClient: APIClientMock!
+    
+    var newUserQueryUpdater: NewUserQueryUpdater<ExtraData>?
+    
+    override func setUp() {
+        super.setUp()
+        env = TestEnvironment()
+        
+        database = try! DatabaseContainerMock(kind: .inMemory)
+        webSocketClient = WebSocketClientMock()
+        apiClient = APIClientMock()
+        
+        newUserQueryUpdater = NewUserQueryUpdater(
+            database: database,
+            webSocketClient: webSocketClient,
+            apiClient: apiClient,
+            env: env.environment
+        )
+    }
+    
+    override func tearDown() {
+        apiClient.cleanUp()
+        
+        AssertAsync {
+            Assert.canBeReleased(&newUserQueryUpdater)
+            Assert.canBeReleased(&database)
+            Assert.canBeReleased(&webSocketClient)
+            Assert.canBeReleased(&apiClient)
+            Assert.canBeReleased(&env)
+        }
+        
+        super.tearDown()
+    }
+    
+    func test_update_called_for_each_query() throws {
```

Comment:
nit: The method name should have `_` between all words. The pattern usually is: `test_something_shouldDoSomething_underCertainConditions`

Reply:
Yep, thank you. I have some issues with test naming 😅
Fixed in 95ec895f